contracts/citizens/DeveloperSkillLevels.sol

```diff
@@ -98,5 +104,19 @@ contract DeveloperSkillLevels {
         skillLevelAverages[developer] = newSkillLevelAverage;
         skillLevelRatings[developer][msg.sender] = rating;
         emit Rated(developer, rating, msg.sender);
+
+        eas.attest(
+            AttestationRequest({
+                schema: easSchemaUID,
+                data: AttestationRequestData({
+                    recipient: developer,
+                    expirationTime: NO_EXPIRATION_TIME,
+                    revocable: false,
+                    refUID: EMPTY_UID,
+                    data: abi.encode(0), // TO DO
+                    value: 0
+                })
+            })
+        );
```

Comment:
Sorry, but I missed another PR:

In Solidity, can you skip this:

```
if (skillLevelAverages[developer] == 0) {
      skillLevelRatingsCount[developer] = 1;
      skillLevelRatingsSum[developer] = ratingInGwei;
  } else {
      skillLevelRatingsCount[developer] += 1;
      skillLevelRatingsSum[developer] += ratingInGwei;
  }

```            
And just do:

```
skillLevelRatingsCount[developer] += 1;
skillLevelRatingsSum[developer] += ratingInGwei;
```

Reply:
@johnmark13 Yes, fixed:  https://github.com/nation3/nationcred-contracts/pull/88/commits/78a0da71a9892407d39c7086514ab79b2caf4d65

Thank you! :-D